src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -278,14 +299,23 @@ export default {
 	position: absolute;
 	bottom: 0;
 	width: 100%;
-	padding: 0 12px 8px 12px;
+	padding: 0 12px 4px 12px;
```

Comment:
variables. and maybe reduce padding on the sides?

```suggestion
	padding: 0 calc(var(--default-grid-baseline) * 2) var(--default-grid-baseline);
```